src/config/keys.ts

```diff
@@ -113,6 +113,10 @@ export const buildItemValidationAndReviewsKey = (id: UUID) => [
   'itemValidationAndReviews',
 ];
 
+export const HOOK_KEYS = {
+  buildItemValidationAndReviewsKey,
```

Comment:
I don't think it's "hook keys" but rather "react query data keys". What do you think of `DATA_KEYS`? And could you add all the keys and not only yours?